go.mod

```diff
@@ -46,35 +46,35 @@ require (
 	golang.org/x/lint v0.0.0-20190409202823-959b441ac422
 	google.golang.org/grpc v1.23.0
 	gopkg.in/yaml.v2 v2.2.4
-	k8s.io/api v0.0.0-20191004102349-159aefb8556b
-	k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783
-	k8s.io/apimachinery v0.0.0-20191004074956-c5d2f014d689
-	k8s.io/apiserver v0.0.0-20191010014313-3893be10d307
-	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/cluster-bootstrap v0.0.0-20190918163108-da9fdfce26bb
-	k8s.io/code-generator v0.0.0-20190912054826-cd179ad6a269
-	k8s.io/component-base v0.0.0-20190918160511-547f6c5d7090
+	k8s.io/api v0.0.0
+	k8s.io/apiextensions-apiserver v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/apiserver v0.0.0
+	k8s.io/client-go v0.0.0
+	k8s.io/cluster-bootstrap v0.0.0
+	k8s.io/code-generator v0.0.0
+	k8s.io/component-base v0.0.0
 	k8s.io/helm v2.14.2+incompatible
-	k8s.io/klog v0.4.0
-	k8s.io/kube-aggregator v0.0.0-20191004104030-d9d5f0cc7532
-	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
-	k8s.io/metrics v0.0.0-20191004105854-2e8cf7d0888c
-	k8s.io/utils v0.0.0-20190801114015-581e00157fb1
+	k8s.io/klog v1.0.0
```

Comment:
I think the `klog` version can also not be bumped if we rely on `v0.16.0` versions.

https://github.com/kubernetes/apimachinery/blob/27d36303b6556f377b4f34e64705fa9024a12b0c/go.mod#L34

Reply:
We use klog v1.0.0 in all of our controllers with no probs, and i think there are no api breaking changes between v0.4.0 and v1.0.0:
https://github.com/kubernetes/klog/compare/v0.4.0...v1.0.0
the v1.0.0 tag was more of a cosmetic action of the k8s authors.

But nevertheless i raised this PR to show some of our findings to simplify the go.mod file and make it more readable/maintainable, because of the hard dependency to a older 1.16 version @ialidzhikov found, this can not be done now. 

Therefore i close this.

Reply:
Thanks anyway. Let's keep it in mind as soon as the controller-runtime uses a later version :)

Reply:
Yes, i will remind us with a followup PR if you don´t mind.